README.md

```diff
@@ -51,3 +51,45 @@ home.packages = with pkgs; [
   brewCasks.marta
 ];
 ```
+
+## Using packages with illegal characters in their name
```

Comment:
Honestly, i don't feel like this needs an entire new section. In my opinion just a line with comment in the initial home-manager example below `brewCasks.marta` would suffice